test/e2e/apibinding/apibinding_test.go

```diff
@@ -304,3 +305,102 @@ func apiexportVWConfig(t *testing.T, kubeconfig clientcmdapi.Config, clusterName
 
 	return rest.AddUserAgent(rest.CopyConfig(config), t.Name())
 }
+
+// TestAPIBindingPermissionClaimCELValidation will validate the permission claims for an otherwise valid APIBinding.
+func TestAPIBindingPermissionClaimCELValidation(t *testing.T) {
```

Comment:
wondering, is there a simple evaluator such that one can unit test CEL expressions? This is really heavy to do it in e2e.

Reply:
updated